packages/framework/presence/src/datastorePresenceManagerFactory.ts

```diff
@@ -38,7 +38,10 @@ class PresenceManagerDataObject extends LoadableFluidObject {
 		if (!this._presenceManager) {
 			// TODO: investigate if ContainerExtensionStore (path-based address routing for
 			// Signals) is readily detectable here and use that presence manager directly.
-			const manager = createPresenceManager(this.runtime);
+			const manager = createPresenceManager(
+				this.runtime,
+				this.runtime.deltaManagerInternal.supportedFeatures,
```

Comment:
this usage should be explained in the design. i think overall we want to avoid reaching from the datastore layer to the loader layer directly, most of those interaction should be mediated through the runtime.